docs/product/components/link-unfurls.html

```diff
@@ -0,0 +1,331 @@
+---
+layout: page
+title: Link unfurls
+description: Link unfurls add rich previews to links included in questions and answers on Stack&nbsp;Overflow. 
+---
+<section class="stacks-section">
+    {% header "h2", "Classes" %}
+    <div class="overflow-x-auto mb32">
+        <table class="wmn4 s-table s-table__bx-simple">
+            <thead>
+                <tr>
+                    <th class="s-table--cell4" scope="col">Class</th>
+                    <th class="s-table--cell3" scope="col">Parent</th>
+                    <th scope="col">Description</th>
+                </tr>
+            </thead>
+            <tbody class="fs-caption">
+                {% for item in link-unfurls.classes %}
+                    <tr>
+                        <th scope="row"><code class="stacks-code">{{ item.class }}</code></th>
+                        <td><code class="stacks-code">{{ item.parent }}</td>
+                        <td class="p8">{{ item.description }}</td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+</section>
+<section class="stacks-section">
+    {% header "h2", "Examples" %}
+    <p class="stacks-copy">Link unfurls, also referred to as oneboxes, are those handy previews that automatically appear when you paste a link into a post. This approach has been popularized by Slack, Twitter, Facebook, etc. Unfurled previews are handy because they give everyone a glimpse of what’s been linked to. Link unfurling should be tailored to fit the content received from each third party. It might be appropriate to use atomic utility classes for further customizing unfurl previews, but Stacks provides a few reusable patterns:</p>
+
+    {% header "h3", "Base style" %}
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="s-unfurl">
+    <div class="s-unfurl--header">
+        <div class="s-unfurl--icon">
+            @Svg.GitHub
+        </div>
+        <div>
+            <div class="s-unfurl--title">
+                <a href="#">…</a>
+            </div>
+            <div class="s-unfurl--details">…</div>
+        </div>
+    </div>
+    <div class="s-unfurl--body">…</div>
+    <div class="s-unfurl--footer">
+        <div class="s-unfurl--url">
+            <a href="#">…</a>
```

Comment:
> It may be appropriate to just recommend that s-link s-link__muted be used inside this component.

I think the component should make this decision so folks don't have to remember to include these classes.

Reply:
@aaronshekey have a preference here? `s-unfurl--line-numbers`?